scripts/npm-publish/npm-publish.go

```diff
@@ -0,0 +1,48 @@
+// Copyright 2025 The Perses Authors
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package main
+
+import (
+	"flag"
+	"fmt"
+	"os/exec"
+	"path/filepath"
+	"regexp"
+
+	"github.com/perses/plugins/scripts/npm"
+	"github.com/sirupsen/logrus"
+)
+
+func main() {
+	tagNamePattern := regexp.MustCompile(`(?m)(.+)/v(\d.\d.\d)`)
```

Comment:
I guess every script running for the release will need this regexp. It will be worth to have a common package where there is this pattern and the function returning the plugin folder name and the version.

Reply:
[I confirm](https://github.com/perses/plugins/pull/70) 😄

Reply:
Hmm actually in the case of CUE module the regexp should be different ("v" must be kept as well as pre-release suffix if provided), so not sure if we can/should factorize stuff here

Or, we make the regexp more complex with more matching groups & subsequent scripts take only what they're interested in

Reply:
the regexp here doesn't care about the pre-released suffix because I forgot about it when I write it. But that should be considered for sure.

Regarding the `v`, I think if you have the correct version, then adding `v` to the version won't be that hard.

I have opened the PR https://github.com/perses/plugins/pull/71 to put in common the logic to get and parse the flag